assets/src/scripts/store/nldi-data.spec.js

```diff
@@ -0,0 +1,209 @@
+import {applyMiddleware, combineReducers, createStore} from 'redux';
+import {default as thunk} from 'redux-thunk';
+
+import {
+    MOCK_NLDI_UPSTREAM_FLOW_FEATURE,
+    MOCK_NLDI_DOWNSTREAM_FLOW_FEATURE,
+    MOCK_NLDI_UPSTREAM_SITES_FEATURE,
+    MOCK_NLDI_DOWNSTREAM_SITES_FEATURE,
+    MOCK_NLDI_UPSTREAM_BASIN_FEATURE
+} from '../mock-service-data';
+
+import {Actions, nldiDataReducer} from './nldi-data';
+
+describe('store/nldi-data module', () => {
+    /* eslint no-use-before-define: 0 */
+    let store;
+
+    beforeEach(() => {
+        store = createStore(
+            combineReducers({
+                nldiData: nldiDataReducer
+            }),
+            {
+                nldiData: {}
+            },
+            applyMiddleware(thunk)
```

Comment:
not sure what this is

Reply:
Basically we are allowing asynchronous redux actions (like the call to retrieve the NLDI data). See https://github.com/reduxjs/redux-thunk